app/2.0/start/quick-tour.md

```diff
@@ -1,21 +1,29 @@
 ---
-title: Quick tour of Polymer
+title: Try Polymer
```

Comment:
If we change the title here, we need to change the title in the navigation as well. And ideally check the other pages that reference that and update any references to "The quick tour" (although I think they mostly say something like "take a quick tour," which is generic enough that it's fine).